examples/unattended_baggage/demo.ipynb

```diff
@@ -0,0 +1,357 @@
+    "\n",
+    "Unattended baggage could simply be an oversight on the part of the owner; or it could be intentionally left behind, containing dangerous materials that pose a threat to public safety. Either way, detection is necessary.\n",
+    "\n",
+    "In this example, we aim to identify unattended baggages and generate results for triggering an alarm in a surveillance video.\n",
+    "A baggage is regarded as unattended if it applies to the circumstances below:\n",
+    "\n",
+    "- The distance to its owner is further than `DISTANCE_THRESHOLD`.\n",
+    "- No one attends the baggage for a period of `t` consecutive seconds.\n",
+    "\n",
+    "Note that in this example, the baggage is not regarded as unattended if there is another person, like a staff, is within the distance threshold."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Environment setup"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Python3.8 is recommended to avoid compatibility issues when installing YOLOX.\n",
+    "\n",
+    "Please download video from [kaggle](https://www.kaggle.com/datasets/szahid405/baggage?select=baggage.mp4) and place it in the same directory as this notebook, aside from running the cell below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# install YOLOX\n",
+    "!git clone https://github.com/Megvii-BaseDetection/YOLOX.git\n",
+    "# download YOLOX pretrained model\n",
+    "!wget https://github.com/Megvii-BaseDetection/YOLOX/releases/download/0.1.1rc0/yolox_x.pth\n",
+    "!cd YOLOX && pip3 install .\n",
+    "# download VQPy, move vqpy/ to root directory for import\n",
+    "!git clone https://github.com/uclasystem/VQPy.git\n",
+    "!mv VQPy/vqpy ./\n",
+    "# install VQPy's dependencies\n",
+    "!pip3 install lap cython_bbox shapely\n",
+    "import vqpy\n",
+    "video_path = \"./baggage.mp4\"\t# or change to wherever you put the video\n",
+    "save_folder = \"./vqpy_outputs\"\n",
+    "detector_model_dir = \"./\""
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For reference, the working directory should look like:\n",
+    "\n",
+    "```text\n",
+    ".\n",
+    "├── VQPy\n",
+    "├── YOLOX\n",
+    "├── baggage.mp4\t# video to query on\n",
+    "├── vqpy\t# make vqpy available for import\n",
+    "└── yolox_x.pth\t# model checkpoint\n",
+    "```"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Unattended baggage query with VQPy"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 1: Define `VObj` type for person\n",
+    "\n",
+    "First, define a `Person` `VObj`. Since we only need two built-in properties:\n",
```

Comment:
#### Define `Person` `VObj`.

Reply:
Since we will only use the built-in properties of `VObj` in `Person`, `Person` can directly inherit `VObjBase`, with no extra property functions defined.
```python
class Person(vqpy.VObjBase):
    pass
```
The built-in properties we will use for `Person` `VObj` are .... . They will be used in the `owner` property of `Baggage` `VObj`, which we will introduce next.